src/testFixtures/java/uk/gov/di/ipv/cri/common/library/client/CommonApiClient.java

```diff
@@ -1,17 +1,35 @@
 package uk.gov.di.ipv.cri.common.library.client;
 
+import software.amazon.awssdk.auth.credentials.AwsSessionCredentials;
+import software.amazon.awssdk.http.SdkHttpFullRequest;
+import software.amazon.awssdk.http.SdkHttpMethod;
+import software.amazon.awssdk.http.SdkHttpRequest;
+import software.amazon.awssdk.http.auth.aws.signer.AwsV4HttpSigner;
+import software.amazon.awssdk.http.auth.spi.signer.SignedRequest;
+import software.amazon.awssdk.regions.Region;
+import uk.gov.di.ipv.cri.common.library.aws.CloudFormationHelper;
 import uk.gov.di.ipv.cri.common.library.util.URIBuilder;
 
 import java.io.IOException;
+import java.net.URI;
+import java.net.URLEncoder;
 import java.net.http.HttpClient;
 import java.net.http.HttpRequest;
 import java.net.http.HttpResponse;
+import java.nio.charset.StandardCharsets;
 
 public class CommonApiClient {
-
     private final HttpClient httpClient;
     private final ClientConfigurationService clientConfigurationService;
 
+    private static final String TEST_RESOURCES_STACK =
+            CloudFormationHelper.getOutput("test-resources", "TestHarnessExecuteUrl");
+    private static final String AWS_ACCESS_KEY_ID = System.getenv("AWS_ACCESS_KEY_ID");
+    private static final String AWS_SECRET_ACCESS_KEY = System.getenv("AWS_SECRET_ACCESS_KEY");
+    private static final String AWS_SESSION_TOKEN = System.getenv("AWS_SESSION_TOKEN");
+    private static final String AWS_SERVICE = "execute-api";
+    private static final Region REGION = Region.EU_WEST_2;
+
```

Comment:
```suggestion

    private static final String TEST_HARNESS_URL =
            CloudFormationHelper.getOutput(TEST_RESOURCES_STACK == null ? "test-resources" : TEST_RESOURCES_STACK, "TestHarnessExecuteUrl");
```

Get the URL using the optional env var value